client/src/views/Dashboard/Dashboard.js

```diff
@@ -64,23 +92,65 @@ const Dashboard = props => {
       return 1
     })
   }
+  const userReviewDetailsArr = []
+  let projects
+  let projectsArr = []
+  if (
+    userSelfReviews &&
+    userSelfReviews.length > 0 &&
+    userReviewDetailsArr.length == 0 &&
+    projectData
+  ) {
+    userSelfReviews.map((review, key1) => {
+      projects = review.project_ids.split(',')
+      projectData.map((item, key) => {
```

Comment:
Loop inside loop, is it required? can we reduce this? There are two hooks for cache. useCallback and useMemo. Can you use that?

Reply:
We are going to update get api so this code we will remove afterwords.

Reply:
Not fixing it .